converter/src/main/java/gov/cms/qpp/conversion/util/FormatHelper.java

```diff
@@ -0,0 +1,37 @@
+package gov.cms.qpp.conversion.util;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class FormatHelper {
+	private static final String DATE_FORMAT = "yyyyMMdd";
+
+	/**
+	 * Formats the date and parses into a {@link LocalDate}.
+	 *
+	 * @param date String to be parsed into a date.
+	 * @return LocalDate parsed from the given string
+	 */
```

Comment:
I went back to this because LocalDateTime.parse does not have a way to accept multiple date patterns.